app/lib/features/space/providers/space_navbar_provider.dart

```diff
@@ -40,7 +40,7 @@ final tabsProvider =
     }
 
     if (appSettings.pins().active()) {
-      final pinsList = await ref.watch(allPinListProvider(spaceId).future);
+      final pinsList = await ref.watch(pinsProvider(spaceId).future);
```

Comment:
Why to use `pinsProvider` here which have bookmarks front sorting calculation. Here we need to check just pins list is empty of not. How about directly using `pinListProvider` or separate provider which given only list item counts or check for empty list only.

Reply:
Doesn't really make any difference...

I just replaced all uses of `allPinListProvider` with the `pinsProvider` in the merge conflict, that's why. But in practice this will not make a difference here.

Reply:
<img width="524" alt="Screenshot 2024-11-08 at 6 08 46 PM" src="https://github.com/user-attachments/assets/fd50e73f-4ffc-4f85-a9f2-ef3eec0c85e9">

As per above code changes, `pinsProvider` contains `priotizeBookmarked` logic which doesn't require here this code use-case.

Agree that it will not make major impact but why to use provider which have additional logic even though direct list provider is available.